modules/gapi/samples/face_detection.cpp

```diff
@@ -180,10 +187,10 @@ std::vector<Face> buildFaces(const cv::Mat& scores,
             Face faceInfo;
             BBox& faceBox = faceInfo.bbox;
 
-            faceBox.x1 = (static_cast<double>(x) * P_NET_STRIDE) / scaleFactor;
-            faceBox.y1 = (static_cast<double>(y) * P_NET_STRIDE) / scaleFactor;
-            faceBox.x2 = (static_cast<double>(x) * P_NET_STRIDE + P_NET_WINDOW_SIZE - 1.f) / scaleFactor;
-            faceBox.y2 = (static_cast<double>(y) * P_NET_STRIDE + P_NET_WINDOW_SIZE - 1.f) / scaleFactor;
+            faceBox.x1 = std::max(0, static_cast<int>((static_cast<float>(x) * stride) / scaleFactor));
```

Comment:
Is this `std::max` equal to:
```
const auto fbx = int(float(x) * stride / sf);
faceBox.x1 = fbx > 0 ? fbx : 0
```
?

Reply:
What's wrong with std::max ? I think it's possible to imagine even more ways to rewrite it.

Reply:
Ok.